homeworks/oleg.borodko_olegborodko/homework_5/task_2.js

```diff
@@ -0,0 +1,19 @@
+function countKey(array) {
+    const object = {};
+
+    for (let i = 0; i < array.length; i++) {
+        if (object[array[i]] == null) {
```

Comment:
`null` is already a value, you need to check if the value is `undefined`... loose check is not good here as well...

actually, I would redesign this using `for...in` or `hasOwnProperty`